app/views/application/_navbar.html.erb

```diff
@@ -1,9 +1,32 @@
-<nav>
-<h3><%= link_to t("nav.home"), root_path %> |
-<% if current_user %>
-  <%= link_to t("nav.signout"), session_path(current_user), method: :delete %> |
-<% else %>
-  <%= link_to t("nav.signup"), new_users_path %> |
-  <%= link_to t("nav.signin"), new_session_path %></h3>
-<% end %>
-</nav>
+<div class="navbar">
+    <% if current_user %>
+      <a><%= link_to t("nav.new_account"), new_users_path %></a>
+      <% if current_user.admin? %>
+      <div class="dropdown">
+        <button class="dropbtn"><%= t("nav.admin") %></button>
+        <div class="dropdown-content">
+          <a><%= link_to t("nav.create_user"), new_users_path %></a><br>
+          <a><%= link_to t("nav.user_index"), admin_users_path %></a>
+        </div>
+      </div>
+      <% end %>
```

Comment:
Indentation definitely looks incorrect in this file

Reply:
Done in sha: a2fa585